.github/workflows/auto-release.yml

```diff
@@ -0,0 +1,52 @@
+name: Auto-Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit_id:
+        description: 'Commit ID'
+        required: true
+        default: 'HEAD'
```

Comment:
I think it would be better to force a specific commit ID to be specified

Reply:
At the moment most of the releases baseline onto HEAD. But need to keep option available for the much less frequent non-HEAD baselined releases